src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -28,15 +31,17 @@ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {GetstreamFeedListenerObject} from '../../../types/hooks/core/getstreamFeedListener/feedListenerObject';
 import {GetstreamMessage, GetstreamWebsocket, MyChannelType} from './websocket/types.d';
 import {InitialStartupAtom} from '../../service/initialStartup';
+import {QueueJobPriority} from '../../core/queue/BaseQueue';
 import {SignedPostNotification} from '../../../types/repo/SignedMessageRepo/SignedPostNotificationData';
 import {getChannelListInfo} from '../../utils/string/StringUtils';
 
 const useCoreChatSystemHook = () => {
-  const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
+  const {localDb, refresh, refreshWithId} = useLocalDatabaseHook() as UseLocalDatabaseHook;
   const {anonProfileId, signedProfileId} = useUserAuthHook();
   const {getAllSignedChannels, getAllAnonymousChannels} = useFetchChannelHook();
   const {getAllSignedPostNotifications, getAllAnonymousPostNotifications} =
     useFetchPostNotificationHook();
+  const {queue} = useDatabaseQueueHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refreshWithId` function is added to the `useLocalDatabaseHook` hook. This seems like a good addition as it allows refreshing specific data by id, which can be more efficient than refreshing all data. However, please ensure that this new function is properly tested and does not introduce any unexpected behavior.

```diff
- const {localDb, refresh} = useLocalDatabaseHook() as UseLocalDatabaseHook;
+ const {localDb, refresh, refreshWithId} = useLocalDatabaseHook() as UseLocalDatabaseHook;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->